src/test/run-make/raw-dylib-inline-cross-dylib/Makefile

```diff
@@ -0,0 +1,27 @@
+# Regression test for calling an inline function that uses a raw-dylib function.
+
+# only-windows
+
+include ../../run-make-fulldeps/tools.mk
+
+all:
+	$(RUSTC) --crate-type dylib --crate-name raw_dylib_test lib.rs -C prefer-dynamic
+	$(RUSTC) --crate-type bin driver.rs -L "$(TMPDIR)" -C prefer-dynamic
+	# Make sure we don't find an import to the function we expect to be inlined.
+	"$(LLVM_BIN_DIR)"/llvm-objdump -p $(TMPDIR)/driver.exe | $(CGREP) -v -e "inline_library_function"
```

Comment:
Can you also add the following:
```
# Also make sure that we *do* find an import to the non-inlined function, so that we
# can be confident that we are actually seeing the import table in the output of `llvm-objdump -p`
"$(LLVM_BIN_DIR)"/llvm-objdump -p $(TMPDIR)/driver.exe | $(CGREP) -e "library_function"
```

With these kinds of tests one is always just one typo away from feeding bogus or an empty string into grep, in which -- of course -- we can't find the text that shouldn't be contained.

Reply:
Done